.claude/hooks/utils/llm/anth.py

```
@@ -0,0 +1,114 @@
+# dependencies = [
+#     "anthropic",
+#     "python-dotenv",
+# ]
+# ///
+
+import os
+import sys
+from dotenv import load_dotenv
+
+
+def prompt_llm(prompt_text):
```

Ruff (0.12.2)

Missing return type annotation for public function `prompt_llm`

(ANN201)